### ISSUE TITLE
App stopped working

### ISSUE THREAD

**Issue by Giszmo** [monlovesmango/astral#34]
"All of a sudden" astral stopped working. After a power outage I spun up quasar normally and connected to localhost:8080 but only got this on the dev console:

```
 22:07:04.323 [Vue warn]: Property "$store" was accessed during render but is not defined on instance. 
  at <QLayout> 
  at <MainLayout onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouterView> 
  at <App> [runtime-core.esm-bundler.js:38](webpack://astral/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js?f781)
22:07:04.323 [Vue warn]: Unhandled error during execution of render function 
  at <QLayout> 
  at <MainLayout onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouterView> 
  at <App> [runtime-core.esm-bundler.js:38](webpack://astral/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js?f781)
22:07:04.324 [Vue warn]: Unhandled error during execution of scheduler flush. This is likely a Vue internals bug. Please open an issue at https://new-issue.vuejs.org/?repo=vuejs/core 
  at <QLayout> 
  at <MainLayout onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< undefined > > 
  at <RouterView> 
  at <App> [runtime-core.esm-bundler.js:38](webpack://astral/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js?f781)
22:07:04.326 Uncaught (in promise) TypeError: _ctx.$store is undefined
    default MainLayout.vue:3
    renderFnWithContext runtime-core.esm-bundler.js:850
    hMergeSlot render.js:25
    setup QLayout.js:226
    renderComponentRoot runtime-core.esm-bundler.js:893
    componentUpdateFn runtime-core.esm-bundler.js:5030
    run reactivity.esm-bundler.js:167
    setupRenderEffect runtime-core.esm-bundler.js:5156
    mountComponent runtime-core.esm-bundler.js:4939
    processComponent runtime-core.esm-bundler.js:4897
    patch runtime-core.esm-bundler.js:4489
    componentUpdateFn runtime-core.esm-bundler.js:5037
    run reactivity.esm-bundler.js:167
    setupRenderEffect runtime-core.esm-bundler.js:5156
    mountComponent runtime-core.esm-bundler.js:4939
    processComponent runtime-core.esm-bundler.js:4897
    patch runtime-core.esm-bundler.js:4489
    componentUpdateFn runtime-core.esm-bundler.js:5107
    run reactivity.esm-bundler.js:167
    callWithErrorHandling runtime-core.esm-bundler.js:155
    flushJobs runtime-core.esm-bundler.js:394
[MainLayout.vue:3](webpack://astral/src/layouts/MainLayout.vue?713b)
```

I cleared the cache, tried Chromium, ... no success. Same error.

**Comment by Giszmo** [comment]
Switching to Branle commits and running those worked. Switching back to Astral and it didn't work. I'm out of ideas what might have changed.

**Comment by monlovesmango** [comment]
hmm thats really weird i haven't touched anything and all is still working for me. can you try opening in a private browser window?

**Comment by Giszmo** [comment]
> can you try opening in a private browser window?

Same stacktrace plus the usual `Failed to open indexedDB, are you in private browsing mode?`

Also now as I switched to master for branle when switching back I got a ton of warnings on `npm install`:

```
$ rm -rf node_modules/
$ npm install
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: vue-virtual-scroller@1.0.10
npm WARN Found: vue@3.2.31
npm WARN node_modules/vue
npm WARN   vue@"^3.0.0" from the root project
npm WARN   6 more (@quasar/app-webpack, @vue/server-renderer, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer vue@"^2.6.11" from vue-virtual-scroller@1.0.10
npm WARN node_modules/emoji-mart-vue-fast/node_modules/vue-virtual-scroller
npm WARN   vue-virtual-scroller@"^1.0.10" from emoji-mart-vue-fast@10.2.2
npm WARN   node_modules/emoji-mart-vue-fast
npm WARN 
npm WARN Conflicting peer dependency: vue@2.7.8
npm WARN node_modules/vue
npm WARN   peer vue@"^2.6.11" from vue-virtual-scroller@1.0.10
npm WARN   node_modules/emoji-mart-vue-fast/node_modules/vue-virtual-scroller
npm WARN     vue-virtual-scroller@"^1.0.10" from emoji-mart-vue-fast@10.2.2
npm WARN     node_modules/emoji-mart-vue-fast
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: vue-resize@0.4.5
npm WARN Found: vue@3.2.31
npm WARN node_modules/vue
npm WARN   vue@"^3.0.0" from the root project
npm WARN   6 more (@quasar/app-webpack, @vue/server-renderer, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer vue@"^2.3.0" from vue-resize@0.4.5
npm WARN node_modules/emoji-mart-vue-fast/node_modules/vue-virtual-scroller/node_modules/vue-resize
npm WARN   vue-resize@"^0.4.5" from vue-virtual-scroller@1.0.10
npm WARN   node_modules/emoji-mart-vue-fast/node_modules/vue-virtual-scroller
npm WARN 
npm WARN Conflicting peer dependency: vue@2.7.8
npm WARN node_modules/vue
npm WARN   peer vue@"^2.3.0" from vue-resize@0.4.5
npm WARN   node_modules/emoji-mart-vue-fast/node_modules/vue-virtual-scroller/node_modules/vue-resize
npm WARN     vue-resize@"^0.4.5" from vue-virtual-scroller@1.0.10
npm WARN     node_modules/emoji-mart-vue-fast/node_modules/vue-virtual-scroller
npm WARN deprecated micro-bip32@0.1.0: Switch to @scure/bip32 for audited version of the lib & updates
npm WARN deprecated micro-base@0.10.2: Switch to @scure/base for audited version of the lib & updates
npm WARN deprecated uuid@3.2.1: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated micro-bip39@0.1.3: Switch to @scure/bip39 for audited version of the lib & updates
npm WARN deprecated @babel/polyfill@7.12.1: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
npm WARN deprecated core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.

added 1082 packages, and audited 1083 packages in 5s

126 packages are looking for funding
  run `npm fund` for details

5 vulnerabilities (2 high, 3 critical)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
```

**Comment by Giszmo** [comment]
Ok, the trick is to use the latest node version. Then it doesn't work.

Now it works ...

```
$ nvm current 
v18.6.0
$ nvm use v14.20.0 
Now using node v14.20.0 (npm v6.14.17)
$ ./node_modules/.bin/quasar dev
```

Please add checks and instructions to the Readme.md on which node version is recommended.

**Comment by monlovesmango** [comment]
ok I think this is fixed now, can you confirm on your end? I updated vuex to 4.0.2 and everything worked fine.

**Comment by Giszmo** [comment]
Sadly the issue remains for v18.6.0

Could you reproduce it for v18 before and it was fixed with the update?

**Comment by monlovesmango** [comment]
yes, I was able to reproduce it, got the exact same error. can you try running `npm update` and see if that resolves it? thats actually what resolved it for me (and it upgraded vuex to v4.0.2). running node version v18.7.0.

**Comment by Giszmo** [comment]
My bad. With npm update it works for me, too.
